packages/react-native-lockdown/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@lavamoat/react-native-lockdown",
+  "version": "0.0.1",
+  "description": "LavaMoat React Native lockdown provides SES lockdown in React Native apps",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LavaMoat/lavamoat.git",
+    "directory": "packages/react-native-lockdown"
+  },
+  "homepage": "https://github.com/LavaMoat/lavamoat#readme",
+  "bugs": {
+    "url": "https://github.com/LavaMoat/lavamoat/issues"
+  },
+  "author": "LavaMoat Team",
+  "license": "MIT",
+  "engines": {
+    "node": ">18.0.0"
+  },
+  "main": "./src/makeGetPolyfills.js",
+  "exports": {
+    ".": "./src/makeGetPolyfills.js",
+    "./repair": "./src/repair.js",
+    "./harden": "./src/harden.js",
+    "./makeGetPolyfills": "./src/makeGetPolyfills.js"
+  },
+  "types": "./types/src/types.d.ts",
+  "files": [
+    "CHANGELOG.md",
+    "src",
+    "types",
+    "!*.tsbuildinfo"
+  ],
+  "scripts": {
+    "test": "ava"
+  },
+  "dependencies": {
+    "@react-native/polyfills": "2.0.0",
+    "ses": "1.12.0"
```

```suggestion
    "ses": "^1.12.0"
```

allows Hermes to init SES after https://github.com/endojs/endo/pull/2723 merged and released as ses@1.12.1 or ses@1.13.0